roles/devscripts/tasks/sub_tasks/_521_gather.yml

```diff
@@ -35,14 +35,6 @@
       vars:
         _nets_data:
           name: "{{ item }}"
-          mtu: "{{ ansible_facts[item]['mtu'] }}"
```

Comment:
Thanks for cleaning this up. I believe there is more work needed to align with `cifmw_networking_definition`

Reply:
Still working my way through... at the moment, I don't have a definitive answer.